data/Sourjaya.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "Sourjaya Das",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Student, Learner soon to be a developer. Open Source enthusiast. Avid football fan.",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/sourjaya_das",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Visit my Github Profile",
+      "url": "https://www.github.com/Sourjaya",
```

Not 100% sure the `www.` is needed here, by the way.
```suggestion
      "url": "https://github.com/Sourjaya",
```